docs/maya/simple_setup.md

```diff
@@ -93,45 +93,60 @@ To create the AdnMuscle deformer with some initial customization, double-click t
   <figcaption><b>Figure 9</b>: AdnMuscle custom creation UI.</figcaption>
 </figure>
 
+In order to add attachments constraints to the muscle we will select the targets (joints, geometries or both), then the muscle with the AdnMuscle applied and finally press the ![add target](images/adn_add_target.png){style="width:4%"} button or *Add Targets* in the AdonisFX menu from the Edit Muscle submenu.
+
+<figure>
+  <img src="images/setup_muscle_add_transform_attachments.png">
+  <figcaption><b>Figure 10</b>: Addition of transform targets to AdnMuscle.</figcaption>
+</figure>
+
+<figure>
+  <img src="images/setup_muscle_add_geometry_attachments.png">
+  <figcaption><b>Figure 11</b>: Addition of geometry targets to AdnMuscle.</figcaption>
+</figure>
+
 Optionally, add Slide On Segment Constraints. This constraint type is recommended for muscles in the limbs of the character to follow better the animation. With the same selection, first the two joints of the rig (shoulder and elbow) and then the muscle geometry, go to AdonisFX Menu > Muscle > *Add Slide On Segment Constraint*.
 
 ### Paint Weights
 
 Once the AdnMuscle deformer is properly created it is possible now to paint its weights to correctly setup the deformer properties. To do so, select the simulated mesh and press the ![paint tool](images/adn_paint_tool.png){style="width:4%"} shelf button or go to AdonisFX Menu > *Paint Tool*.
 
-Start by painting attachment weights, painting the influence for each attachment by selecting the corresponding attachment from the list and painting its desired influence.  
+Start by painting attachment weights, painting the influence for each target by selecting the corresponding target from the list and painting its desired influence.
+
+> [!NOTE]
+> In this example the influence painting for both types of attachment constraint is shown. The different types of attachments can be used together or not depending on the needs of the simulation and the rig setup.
 
 <figure>
-  <img src="images/setup_muscle_3.png"> 
-  <figcaption><b>Figure 10</b>: Attachment influence for joint 1.</figcaption>
+  <img src="images/setup_muscle_paint_transform_attach.png">
+  <figcaption><b>Figure 12</b>: Transform Attachment influences.</figcaption>
```

Comment:
```suggestion
  <figcaption><b>Figure 12</b>: Transform Attachment influences (joints and bones).</figcaption>
```

Reply:
I've added your suggestion but should we add "bones"?

Reply:
Maybe instead of bones we could add "locators"

Reply:
Like it!